spec/restaurant_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'restaurant'
+
+describe Restaurant do
+  subject { [{ name: "Boneless Spare Ribs", price: 9 }] }
```

Comment:
Maybe you can copy and paste all menu items to ensure test coverage is higher - but I feel like there is an easier way to test - just not too sure.